### PR TITLE
Add command to create new block at top of buffer

### PR DESCRIPTION
**Pull request by fabjan**
## Praise

Hey this is a really cool tool. I often have to find some arbitrary text editor, and then my files, and then pick some scratch file etc. when I want to jot down notes.

## What's this PR?

Adds a shortcut to create a new block at the top of the buffer, and move the cursor there.

## Why?

When I collect notes I like to put the new things up top, the longer notes linger the less important they usually get, so in my view it's nice if they slowly just flow down out of sight lower in the document. I understand this is a preference and not everyone wants things the same, but I figured I could share my local changes so I gave it a separate shortcut here and opened a PR.

I totally get if the plan is to not add a bunch of features.

### CODE REVIEW

File: src/editor/block/commands.js [hunk 1]
```diff
@@ -47,6 +47,25 @@ export const addNewBlockAfterCurrent = ({ state, dispatch }) => {
     return true;
 }
 
+export const addNewBlockUpTop = ({ state, dispatch }) => {
+    if (state.readOnly)
+        return false
+    const block = state.facet(blockState)[0]
```
Comment (fabjan) [inline src/editor/block/commands.js:53]: It seems there is always at least one block in the buffer so this should work. I could not make it break by fiddling around deleting everything in the editor etc.

**Comment by heyman** [comment]
It may make more sense to have it add the block before the current one. That's how `Ctrl/Cmd + Enter` works, so it would be analogous.

**Comment by fabjan** [comment]
I see #85 is seeing some action so I'm not working on this one. Keeping it here for the other PR author to reference should they want.

**Comment by heyman** [comment]
Fixed by #85

**Pull request closed without merging**